update unity docs and added react native release notes (#15834)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx

  style={{ maxWidth: '70%' }}
/>

> 4. If the Podfile is not being used for iOS dependency management, you can proceed with the following steps.

Reviewer: ```suggestion
4. If you're not using the Podfile for iOS dependency management, follow these steps:
```

>    1. Download and unzip the New Relic XCFramework SDK
>     Download the latest iOS agent from our [iOS agent release notes](https://docs.newrelic.com/docs/release-notes/mobile-release-notes/ios-release-notes)
>    2. Add the New Relic XCFramework to your Xcode project
>      Unzip the SDK download, drag the “NewRelicAgent.xcframework” folder from the Finder into your Xcode project (dropping it onto your Targets Frameworks pane). Select “Embed & Sign” under the Embed column.
> 
> 
> And you're done! You should start seeing data in New Relic within 5 minutes.

Reviewer: ```suggestion
   1. Download and unzip the New Relic XCFramework SDK (iOS agent). You can get the latest iOS agent from our [iOS agent release notes](/docs/release-notes/mobile-release-notes/ios-release-notes).
   2. Add the New Relic XCFramework to your Xcode project:
         1. Unzip the SDK download.
         2. From your Finder, drag the `NewRelicAgent.xcframework` folder into your Xcode project, placing it onto your Targets Frameworks pane. 
         3. Under the **Embed** column, click **Embed & Sign**. 

And you're done! You should start seeing data in New Relic within 5 minutes.
```

</table>

> ## Query C# Errors(Fatal/Non Fatal).
> 
> C# errors and handled exceptions can be seen in the `Handled Exceptions` tab in New Relic One. You will be able to see the event trail, attributes, and stack trace for each C# error recorded.
> 
> You can also build a dashboard for these errors using this query:
> 
> ```sql
> SELECT * FROM MobileHandledException SINCE 24 hours ago
> 

Reviewer: ```suggestion
## Query C# Errors [#query-errors]

In the New Relic UI, you can view your fatal and non-fatal C# errors and handled exceptions, including details like event trails, attributes, and stack traces for each recorded error.

<img
  title="Mobile Handled Exceptions"
  alt="Mobile Handled Exceptions"
  src={mobileHandledExceptionsSummary}
/>

To view your errors:
1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile**.
2. Select your Unity app.
3. Click **Exceptions > Handled exceptions**. 

You can also build a dashboard for these errors using this query:

```sql
SELECT * FROM MobileHandledException SINCE 24 hours ago
```

</table>

> ## Query C# Errors [#query-errors]

Reviewer: ```suggestion
## Query C# errors [#query-errors]
```

> ## Query C# errors [#query-errors]

Reviewer: ```suggestion
## Monitor C# errors [#query-errors]
```
